go/runtime/registry/host.go

```diff
@@ -87,6 +87,22 @@ func (n *RuntimeHostNode) ProvisionHostedRuntimeComponent(comp *bundle.ExplodedC
 	return nil
 }
 
+// RemoveHostedRuntimeComponent removes the given runtime component.
```

Comment:
This allows only removing ROFL components right? Maybe repeat the comment from `RemoveComponent`: `// Attempting to remove the RONL component will result in an error.`